doc/ecs/components/ai_component.md

```diff
@@ -3,20 +3,49 @@ title: "AI Component"
 ---
 
 ## Wofür
-- Wofür braucht es diese Comonent?
-- Was macht es? 
+
+Das `AIComponent` wird verwendet, um eine Entität verhalten zu geben. Dabei gibt es zwei Zustände, `idle` und `fight`,
+welche durch die `transitionAI` bestimmt werden.
 
 ## Aufbau
 
-- UML
-- Aufbau Erklären
+![UML Diagram](../img/aiComponent.png)
```

Comment:
Werden diese Bilder nur hier in `docs/ecs/components/` benötigt oder teilen sich verschiedene Dokumente auf verschiedenen Ebenen die Bilder? 

Falls nicht, sollte der Ordner `img` (oder zumindest die relevanten Abbildungen) bitte auf diese Ebene verschoben werden, also `docs/ecs/components/img/`. Damit ändert sich der Link: `![UML Diagram](img/aiComponent.png)`.

Reply:
Aktuell gibt es nur Bilder in den Docs der Components. Ich denke aber das es auch Bilder in den anderen Docs geben wird. Ich habe nur die vorhandene Struktur verwendet 😉

Reply:
dann lege bitte einen passenden bilderordner auf der aktuellen ebene an und nutze den hier.